WhaleSpotting/react-app/src/styles/Navbar.scss

```
@@ -0,0 +1,51 @@
+@import './constants.scss';
+
+.header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    background-color: $primaryColour;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $headerHeight;
+    z-index: 10;
+}
+
+.nav-bar {
+    display: flex;
+    width: 100%;
+    max-width: 1024px;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.navbar-link {
+    margin: 0% 5%;
```

You do not need it to be 0%, it can just be 0
```suggestion
    margin: 0 5%;
```